src/pages/index.astro

```diff
@@ -140,5 +118,14 @@ const cards = [weatherCard, calculatorCard, realEstateCard, forestCard];
       </div>
     </section>
   </main>
-  <script src="../scripts/modal.js"></script>
+  <Modal modalContent={modalContent}>
```

Comment:
Do we actually need to pass both props and slots? I think we can the props we pass

Reply:
pass only props